cotainr/util.py

```diff
@@ -160,3 +166,22 @@ def _print_and_capture_stream(*, stream_handle, print_dispatch):
         captured_stream.append(line)
 
     return captured_stream
+
+
+def answer_yes(input_text):
+    """
+    Print text and compare the input given to "yes"
+
+    Parameters
+    ----------
+    input_text : str
+        a string to be printed for verification by the user
+
+    Returns
+    -------
+    answer_is_yes : boolean
```

Comment:
`boolean` is not a Python type.
```suggestion
    answer_is_yes : bool
```